python/eos/observable_TEST.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+import eos
+
+class StaticMethodTests(unittest.TestCase):
+
+    def test_assert_valid_name_0(self):
+
+        "name is valid"
```

Comment:
Is this supposed to be a comment?

Reply:
Well, a description of the test instead of a long function name. The strings show up in the (verbose) test report.

Reply:
Understood, thanks!